styleguide/components/11-base/forms/_forms.scss

```diff
@@ -115,11 +112,7 @@ $field-icon-padding: 2.5rem;
   border: 1px solid;
   border-radius: border-radius('radius-1');
   font-family: $default-font-family;
-  font-size: 1rem;
+  font-size: .7rem;
```

Comment:
Gaan we op mobile dan niet onder de 16px (.7rem is toch 14px)? => geeft zooming issue op iOS